golos.publication/golos.publication.abi

```diff
@@ -789,6 +797,52 @@
         }
     ]
   },
+  {
+    "name": "deletevotes",
+    "base": "",
+    "fields": [
+        {
+            "name": "message_id",
+            "type": "int64"
+        },
+        {
+            "name": "author",
+            "type": "name"
+        }
+    ]
+  },
+  {
+    "name": "recipient",
+    "base": "",
+    "fields": [
+        {
+            "name": "to",
+            "type": "name"
+        },
+        {
+            "name": "quantity",
+            "type": "asset"
+        },
+        {
+            "name": "memo",
+            "type": "string"
+        }
+    ]
+  },
+  {
+    "name": "payto",
```

Comment:
It's seems that `payto` action is unnecessary for posting smart-contract. I can't imagine scenario when witnesses should call `payto` method.